### DISCUSSION TITLE
Table editor not loading, showing 'error' (probably due to 500 error)

### DISCUSSION THREAD

**Discussion by PsychotherapistSam** [supabase/supabase#500] [Questions] [answered]
Hey! I have tried supabase for a few days now and am really happy with it. Since a day or two the table editor doesn't load and just displays "Error". 
After doing some investigation the get request to `https://app.supabase.io/api/props/project/<myproject>/table` returns with an error 500 after 30 seconds of loading. This affects the table editor as well as the table settings.
Everything else works as expected (I access the table(s) directly via PostgreSQL).
I don't know whether this is an issue with my table or any configuration I have changed. 

I hope this is the right place to ask :)

**Comment by inian** [comment, marked as answer]
Hi @PsychotherapistSam, I took a look at your project and looks like the realtime server was using up a lot of CPU and it was hitting the CPU limits we had allocated to your project. For now, I have upgraded your project to a bigger server which should solve the issue you were having. 

Are you currently using the realtime functionality of Supabase? If not, you could disable the realtime functionality by running this 
`BEGIN;
DROP PUBLICATION supabase_realtime;
CREATE PUBLICATION supabase_realtime;
COMMIT;`

**Reply by PsychotherapistSam** [reply]
Hi @inian, 

I was not using the realtime functionality and have now disabled it. Would there be an easy way to reactivate it later (if I needed it)?
Thanks a lot for the help, I appreciate it!

Greetings,
Sam

**Reply by inian** [reply]
Great! I can also see that things are working correctly after realtime has been disabled. 

You can always turn realtime back on for individual tables using 
`ALTER PUBLICATION supabase_realtime ADD TABLE table1, table2`

**Reply by PsychotherapistSam** [reply]
Thank you! :)

**Comment by fortydegrees** [comment]
I also have this issue, along with not being able to access any of my database via the API. I first noticed when the app I was developing would time out.

Similarly, when I go to view the table in the web app, it just stays on 'loading tables' for 30 seconds and then says 'error connecting to API'. I can see the existence of the tables in the 'Database' tab, and the dashboard analytics in 'Home' work, but 'Tables' fails to load/display, and I can't access via API.

I initially thought supabase might be down, but my other projects work fine. 

I tried disabling realtime functionality via the mentioned query above, but that didn't resolve the issue.

**Reply by fortydegrees** [reply]
It seems related to this: https://github.com/supabase/supabase/issues/778

That is, I sent a request to insert what was probably a large amount of data. It was tweets I pulled from twitter's API, so nothing crazy. Almost definitely below 50Mb.

**Reply by fortydegrees** [reply]
For what it's worth, restarting the server (via 'Settings'->'Restart server') fixed the issue. Didn't know this was available but makes sense now. Leaving the solution here as this is the thread I was brought to while googling.

**Comment by DiegoSMesquita** [comment]
Boa Tarde, estou com o erro também meu app ao solicitar /rest/v1 ele da o erro 500 estou usando o SUPABASE SELFHOST no meu servidor linux.
ele já está online pelo domínio consigo abrir mais quando coloco /rest/v1 no final do domínio ele da erro 404 quando abro direto no navegador porém no localhost tentando via comando ele funciona normal.
